staff/eduardo-sanchez/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+thums.db
```

Comment:
typo in code. to fix this change line 2 to:
```
thumbs.db
```

Reply:
Thanks !! 👍